ocs_ci/ocs/resources/pod.py

```diff
@@ -300,6 +300,58 @@ def run_git_clone(self):
         assert wl.setup(), "Setup up for git failed"
         wl.run()
 
+    def install_packages(self, packages):
+        """
+        Install packages in a Pod
+
+        Args:
+            packages (list): List of packages to install
+
+        """
+        if isinstance(packages, list):
+            packages = ' '.join(packages)
+
+        cmd = f"yum install {packages} -y"
+        self.exec_cmd_on_pod(cmd, False)
```

Comment:
I would suggest to add the parameter name for the `False` parameter, for better readability, without checking the method definition:
```suggestion
        self.exec_cmd_on_pod(cmd, exec_cmd_on_pod=False)
```
(similarly on other places below)

Reply:
Done